submissions/korotushak96/memory-pair-game/js/script.js

```diff
@@ -0,0 +1,106 @@
+const gameInit = ()=>{
+    const SCENE = document.querySelector('.scene');
+    const POKEMONS = [
+        {
+            id: 0,
+            src: 'img/bulbasaur.png'
+        },
+        {
+            id: 1,
+            src: 'img/charmander.png'
+        },
+        {
+            id: 2,
+            src: 'img/eevee.png'
+        },
+        {
+            id: 3,
+            src: 'img/lapras.png'
+        },
+        {
+            id: 4,
+            src: 'img/pikachu.png'
+        },
+        {
+            id: 5,
+            src: 'img/squirtle.png'
+        }
+    ];
+    
+    let contentArray = [...POKEMONS,...POKEMONS];
+    contentArray.sort(function() { return 0.5 - Math.random() });
+
+    let pokeHTML = '';
+    contentArray.forEach(({id, src}, index)=>{
+        pokeHTML+=`
+            <div class="card is-flipped " data-id="${id}" data-index='${index}'>
+                <div class="card__face card__face--front">
+                    <img src='${src}'>
+                </div>
+                <div class="card__face card__face--back ">
+                    <img src='img/back.jpg'>
+                </div>
+            </div>
+        `
+    });
+    
+    SCENE.insertAdjacentHTML('afterbegin', pokeHTML);
+    let firstCard = -1;
+    let secondCard = -1;
+    let disabed = 0;
+    const cards = [...document.querySelectorAll('.card')];
+    const backSide = document.querySelectorAll('.card__face--back')
+    const flipCard = (index)=>{
+        let activeCard = cards.find(card => index===card.dataset.index);
+        if (activeCard) activeCard.classList.toggle('is-flipped')
+    };
+    
+    const changeStorage = (index)=>{
+        if (firstCard < 0)  firstCard = index;
+        else if (secondCard < 0) secondCard = index; 
+        if (firstCard>=0 && secondCard >=0) {
+            backSide.forEach(card => card.classList.add('block'));
+            checkEq(firstCard, secondCard);
+        };  
+    };
+    
+    const checkEq = (first, second) => { 
+        let cardOne = cards.find(card => first===card.dataset.index).dataset.id;
+        let cardTwo = cards.find(card => second===card.dataset.index).dataset.id;
```

Comment:
Read about Prettier and install it for nice formatting your code

Reply:
Thanks. Nice feature